packages/tooltips/src/styled/arrow-styles.ts

```diff
@@ -0,0 +1,193 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import { css, ThemeProps, DefaultTheme } from 'styled-components';
+import math from 'polished/lib/math/math';
+import stripUnit from 'polished/lib/helpers/stripUnit';
+import { IStyledTooltipProps } from './StyledTooltip';
+
+export const arrowStyles = ({
+  theme,
+  hasArrow,
+  size
+}: IStyledTooltipProps & ThemeProps<DefaultTheme>) => {
+  if (!hasArrow) {
+    return '';
+  }
+
+  const arrowSize = `${stripUnit(theme.fontSizes.sm) / stripUnit(theme.fontSizes.md)}em`;
+  const arrowTransform = 'rotate(45deg)';
+  const arrowPosition = math(`${arrowSize} / -2`);
+  const bottomClipPath =
+    'polygon(100% 0, calc(100% - 1px) 0, 0 calc(100% - 1px), 0 100%, 100% 100%)';
+  const leftClipPath = 'polygon(0 100%, 100% 100%, 100% calc(100% - 1px), 1px 0, 0 0)';
+  const rightClipPath = 'polygon(100% 0, 100% 100%, calc(100% - 1px) 100%, 0 1px, 0 0)';
+  const topClipPath = 'polygon(100% 0, 100% 1px, 1px 100%, 0 100%, 0 0)';
+
+  let arrowFontSize = `${theme.space.base * 1.75}px`;
+
+  if (size === 'extra-large') {
+    arrowFontSize = `${theme.space.base * 4}px`;
+  } else if (size === 'large') {
+    arrowFontSize = theme.fontSizes.sm;
+  }
```

Comment:
Move this into `StyledTooltip` as it is directly associated with that component as opposed to this more generic arrow styling.

I guess the big question is how we can pull this arrow styling out into something that can be re-used across components? Feels like we would be better served if this moved to `theming` or (blerg) `utilities`.

Reply:
I have moved the size-specific logic to the `StyledTooltip` component.

> how we can pull this arrow styling out into something that can be re-used across components?

From our offline conversation, with the differences between arrow (specifically animation) don't think we should try and solve this reuse issue until we tackle that package.